### PR TITLE
build: tailwind autosort

### PR DESCRIPTION
**Pull request by cabreraalex**
# Description

Automatically sort tailwind classes

### CODE REVIEW

**Comment by cabreraalex** [comment]
Having some issues with vscode and the npm commands doing different things

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,12 +1,11 @@
 {
   "python.defaultInterpreterPath": ".venv/bin/python",
-  "autoDocstring.docstringFormat": "google",
```
Comment (Sparkier) [inline .vscode/settings.json:3]: Shouldn't we keep this?

File: frontend/.prettierrc [hunk 2]
```diff
@@ -3,13 +3,5 @@
...
 	"printWidth": 100,
-	"plugins": ["prettier-plugin-svelte"],
-	"overrides": [
-		{
-			"files": "*.svelte",
-			"options": {
-				"parser": "svelte"
-			}
-		}
-	]
```
Comment (Sparkier) [inline frontend/.prettierrc:14]: Sure this is not needed?

File: frontend/package.json [hunk 3]
```diff
@@ -40,7 +40,7 @@
 		"eslint-plugin-svelte": "^2.33.0",
 		"postcss": "^8.4.29",
 		"prettier": "^3.0.3",
-		"prettier-plugin-svelte": "^3.0.3",
+		"prettier-plugin-tailwindcss": "^0.5.6",
```
Comment (Sparkier) [inline frontend/package.json:43]: Don't we need both plugins?

File: frontend/src/app.postcss [hunk 4]
```diff
@@ -1,3 +1,6 @@
+/* Write your global styles here, in PostCSS syntax */
+
+@tailwind base;
```
Comment (Sparkier) [inline frontend/src/app.postcss:3]: What's going on here?

**Review by Sparkier — CHANGES_REQUESTED** [review]

File: frontend/package.json (hunk 3 above)
Comment (cabreraalex) [reply frontend/package.json:43]: the svelte one comes bundled in this one

File: .vscode/settings.json (hunk 1 above)
Comment (cabreraalex) [reply .vscode/settings.json:3]: I stopped using it and just use copilot so it showed not used but yes can keep probably better.

File: frontend/src/app.postcss (hunk 4 above)
Comment (cabreraalex) [reply frontend/src/app.postcss:3]: I think this is supposed to be postcss, I re-ran the setup for tailwind. Also updated import in main app component.

File: frontend/.prettierrc (hunk 2 above)
Comment (cabreraalex) [reply frontend/.prettierrc:14]: it was added back

**Comment by cabreraalex** [comment]
This is working consistently across VSCode, CLI, and GH Actions for me

**Review by Sparkier — APPROVED** [review]

**Pull request merged**